### DISCUSSION TITLE
240325

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#75] [팀회의]

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
캘린더 리팩토링, 데이터 연동
## 오늘 할 일
일정생성모달, 그룹생성모달 부분 시간 설정 ui&기능 만들기 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
https://github.com/codeit-part4-8team-project/main/pull/74
- 팀원 페이지?

## 어제 한 일

- 팀 페이지 탭 기능 구현
- Outlet 사용하여 공통 레이아웃별로 나누어 경로 수정
- 유저 자유게시판 페이지 추가 / 팀 세부 페이지들 추가

## 오늘 할 일
- 사이드바 / 자유게시판 / 칸반보드 api
- 공지사항 ui / 자잘한 ui 수정 

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항
뭐할까요...?, PR설명...?
## 어제 한 일
주말에 시골에 갔다오느라 일을 못했습니다
## 오늘 할 일
디자인 업데이트 반영.
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
x
## 어제 한 일
api연동, useAxios격파하기
## 오늘 할 일
디자인 업데이트 반영, 노션에 정리해서 알려주기, 새로생긴 모달 만들기
## 오늘의 일정 (코어 타임)
오열코!
## 멘토님 질문 사항
x
